backend/src/main/resources/application.yaml

```diff
@@ -1,9 +1,8 @@
 spring:
   datasource:
-    url: ${POSTGRES_HOST:postgres_fqdn}?sslmode=require
-    username: ${DB_USERNAME:postgres_user}
-    password:  ${POSTGRES_USER:postgres_password}
-    name: ${POSTGRES_DB:postgres_db_name}
+    url: jdbc:postgresql://${POSTGRES_HOST:localhost}:${DB_PORT:5432}/${POSTGRES_DB:reportvision}?sslmode=${SSL_MODE:disable}
```

Comment:
Do we want to `enable` ssl_mode to enforce SSL encryption for this connection?

Reply:
I was thinking we can pass SSL_MODE as a argument(enabled,disabled) but kept it disabled by default for connecting to local host. Would this work?

Reply:
It looks like @jonchang and @derekadombek may have tested this and it works as disabled. `SSL_MODE` enabled adds additional security.  However, I will defer to the majority decision of the team on whether it should be enabled at this time or before we hand it off.

Reply:
ok we can set it as enabled by default then.